src/App.css

```diff
@@ -644,7 +679,7 @@ body,
   animation-name: fly-in-right;
   animation-duration: 1s;
   animation-fill-mode: forwards;
-  z-index: 1;
+  z-index: 2;
```

Comment:
I had to increase the z-index on the orbit_closeup_container so the buttons on it wont be blocked by the trigger divs.